packages/ag-charts-website/src/content/docs/doughnut-series/index.mdoc

```diff
@@ -2,104 +2,107 @@
 title: 'Doughnut Series'
 ---
 
-A Doughnut Series displays categorical data proportions and allows multiple series.
+A Doughnut Series is a variation of a Pie Series, and allows displaying multiple datasets in a single chart.
 
-Pie series can be used to create a doughnut chart by using the `innerRadiusOffset` config.
+## Simple Doughnut
+
+{% chartExampleRunner title="Doughnut Chart" name="simple-doughnut" type="generated" /%}
+
+To create a Doughnut Series, use the `pie` series type and specify an `innerRadiusRatio`.
 
 ```js
 series: [
     {
         type: 'pie',
-        calloutLabelKey: 'os',
-        angleKey: 'share',
-        innerRadiusOffset: -70,
+        calloutLabelKey: 'asset',
+        angleKey: 'amount',
+        innerRadiusRatio: 0.7,
     },
 ];
 ```
 
-The config specifies the offset value from the maximum pie radius which all pie sectors use by default (the maximum pie series radius is determined automatically by the chart depending on the chart's dimensions). `-70` in the snippet above means the inner radius of the series should be 70 pixels smaller than the maximum radius.
+The `innerRadiusRatio` should be a value between `0` and `1` and defines the radius of the inner circle as a ratio of the outer radius of the chart.
 
-{% chartExampleRunner title="Doughnut Chart" name="doughnut-chart" type="generated" /%}
+## Inner Labels
 
-Alternatively a doughnut chart can be created by using the `innerRadiusRatio` property. A value between `0.0` and `1.0` should be assigned:
-
-```diff
-series: [{
-    type: 'pie',
-    labelKey: 'os',
-    angleKey: 'share',
-+   innerRadiusRatio: 0.75,
-}]
-```
+{% chartExampleRunner title="Text Inside a Doughnut Chart" name="text-inside-doughnut" type="generated" /%}
 
-### Additional Doughnut Labels
+The `innerLabels` property can be used to put several lines of text in the space inside a Doughnut Series.
 
-The `innerLabels` property can be used to put several text lines inside a doughnut chart.
-The colour of the doughnut's centre can be changed by using `innerCircle`.
+The colour of the centre area can be changed by using `innerCircle`.
 
 ```js
 series: [{
-    ...
-    innerLabels: [
-        { text: '85%', fontSize: 48, color: 'green' },
-        { text: 'Coverage', margin: 4 }
-    ],
-    innerCircle: {
-        fill: 'lightgreen',
-    }
-}]
+        ...
+        innerLabels: [
+            {
+                text: 'Total Investment',
+                fontWeight: 'bold',
+            },
+            {
+                text: '$100,000',
+                margin: 4,
+                fontSize: 48,
+                color: 'green',
+            },
+        ],
+        innerCircle: {
+            fill: '#c9fdc9',
+        },
+    }]
```

Comment:
The indentation here is a bit off. The `...` trips up the formatter. Same issue in a few other places too.

Reply:
I've commented the `...`, formatted and then uncommented it again.
Seems to have worked.